libs/sdk-ui-dashboard/.dependency-cruiser.js

```diff
@@ -113,6 +113,7 @@ options = {
             "src/model/events/widget.ts",
             "src/presentation/constants",
             "src/presentation/dashboardContexts",
+            "src/presentation/dashboardContexts/DashboardComponentsContext.tsx",
```

Comment:
This should not be necessary, you are probably importing DashboardComponentsContext.tsx directly somewhere, not using the barell in dashboardContexts.

Reply:
Removed as the ThrowMissingComponentError does not need to be exported.